okd-rpms.repo

```diff
@@ -0,0 +1,9 @@
+[okd-rpms]
+name=OKD RPMs
+failovermethod=priority
```

Comment:
the `failovermethod` keyword is deprecated (see also: https://github.com/coreos/fedora-coreos-config/pull/1048)

Reply:
Filed https://github.com/openshift/okd-machine-os/issues/137